freshmaker/pyxis.py

```diff
@@ -178,21 +178,30 @@ def get_latest_bundles(self, index_images):
 
         return latest_bundles
 
-    def get_manifest_list_digest_by_nvr(self, nvr):
+    def get_manifest_list_digest_by_nvr(self, nvr, must_be_published=True):
```

Comment:
maybe `published=True` is clear enough?

Reply:
I decided against it because `published=False` insinuates that we don't want those from published repositories, but really, we just want to make the safety checks less strict.